pkg/config/config.go

```diff
@@ -146,6 +146,9 @@ type ReceiverConfig struct {
 	Fields            map[string]interface{} `yaml:"fields" json:"fields"`
 	Components        []string               `yaml:"components" json:"components"`
 
+	// ExcludeKeys settings
+	ExcludeKeys []string `yaml:"excludeKeys" json:"excludeKeys"`
```

Comment:
лучше все в одном формате держать

```suggestion
	ExcludeKeys []string `yaml:"exclude_keys" json:"exclude_keys"`
```

Reply:
Спасибо! Исправил